tests/test_api_versions.py

```
@@ -0,0 +1,104 @@
+
+
+try:
+    from rest_framework.test import APIClient
+    HAS_DRF = True
+except Exception:
+    HAS_DRF = False
+
+
+@pytest.mark.django_db
+class TestProjectUrls:
+    def test_schema_url_resolves(self, settings: Any) -> None:
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `settings`

(ANN401)